dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/CpuInfo.java

```diff
@@ -253,7 +252,7 @@ public static CpuInfoAccessor createCpuInfoAccessor() {
             return new NullCpuInfoAccessor();
         }
         try {
-            if (JavaInfo.vendor() == Vendor.IBM) {
+            if (JavaInfo.isAvailable("com.ibm.lang.management.OperatingSystemMXBean")) {
```

Comment:
Why not just use `com.ibm.security.auth.module.Krb5LoginModule` to detect whether it is IBM Java?

Reply:
Because we don't _really_ care if it is an IBM Java ... we care if the function provided by `com.ibm.lang.management.OperatingSystemMXBean` is available